src/javascripts/ng-admin/Crud/list/ListController.js

```diff
@@ -47,9 +47,16 @@ define(function () {
 
         this.ReadQueries
             .getAll(this.view, page, true, this.search, this.sortField, this.sortDir)
-            .then(function (nextData) {
+            .then(function (response) {
                 progression.done();
-                self.entries = self.entries.concat(nextData.entries);
+                self.dataStore.mapEntries(
```

Comment:
You need to retrieve references before mapping and set entries to the datastore.